src/Spago/Command/Resolution.purs

```diff
@@ -0,0 +1,45 @@
+module Spago.Command.Resolution where
```

Comment:
This module does not follow our naming convention for this folder, since there's no command called `resolution`. Since this stuff is only used in one place I would like to inline it in the `let` block in line 629 in `Spago.Command.Fetch`